tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -277,6 +271,11 @@ static cl::opt<SPIRV::BuiltinFormat> SPIRVBuiltinFormat(
         clEnumValN(SPIRV::BuiltinFormat::Global, "global",
                    "Use globals to represent SPIR-V builtin variables")));
 
+static cl::opt<bool> SPIRVUseLLVMSPIRVBackendTarget(
+    "spirv-use-llvm-backend-target",
+    cl::desc("Convert LLVM to SPIR-V using the LLVM SPIR-V Backend target"),
```

Comment:
Do we want to add an extra sentence to say what will happen if SPIR-V target is not found? Also, add a quick phrase to specify default setting.

Reply:
Sure